src/samples/UnAuthChildBenefitsClaim/index.tsx

```diff
@@ -49,6 +44,16 @@ function initTimeoutUnauth(setShowTimeoutModal) {
   }, milisecondsTilWarning);
 }
 
+// Sends 'ping' to pega to keep session alive and then initiates the timout
+function staySignedIn(setShowTimeoutModal, refreshSignin = true) {
+  if (refreshSignin) {
```

Comment:
if refreshSignin  is always true, why do we need to have the if?

Reply:
I've updated that, as yes it doesn't seem to be used anywhere.